docs/ref-database-util.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Database util
+category: reference
+order: 99
+---
+
+The so-called `Database util` is a tool for managing database entries created by abapGit. You can access the tool via the tools icon in the top right corner of the home page of the abapGit report > choose first menu item `Database util`.
+
+It's possible to edit database entries of type
+
+- `USER` and
+- `REPO`.
+
+An `USER` entry does contain meta information like the favorites of an user and its repository configuration e. g. name and email address for Git. This is an example of an `USER` entry:
```

Comment:
Small spelling/grammar suggestions (not a native speaker so others may correct me as well):
```suggestion
A `USER` entry contains meta information like the favorites of a user and their repository configurations e. g. name and email address for Git. This is an example of a `USER` entry:
```